### PR TITLE
Resolve #66 Update list of Active Member Priveleges

### LINKED ISSUE #66: 4.B.4 Bulleted List Changes
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
- Keep item 1
- Remove item 2
- Reword item 3
- Maybe delete item 4
- Remove item 5
- Combine items 6 & 7

### PR DESCRIPTION
**Pull request by esoccoli**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Fixed bulleted list in 4.B.4 based on discussion during Tuesday's meeting

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -1043,7 +1040,8 @@
 This organization shall not discriminate on the basis of sex, race, color, sexual orientation, gender identity and gender expression, religion, age marital state, national origin, disability or veteran status.
 This policy will include but is not limited to, recruiting, membership, organization activities or opportunities to hold or run for club office.
 
-\asection{Statement of Compliance with University Regulations}
+\asection{Statement of Compl
+iance with University Regulations}
```
Comment (mxmeinhold) [inline constitution.tex:1044]: Why is this on a new line? Document standard is a sentance per line

**Review by mxmeinhold — CHANGES_REQUESTED** [review]

File: constitution.tex (hunk 1 above)
Comment (esoccoli) [reply constitution.tex]: Fixed it

File: constitution.tex [hunk 2]
```diff
@@ -198,12 +198,9 @@
 Active Members receive the right to:
 \begin{itemize}
 	\item Vote on all issues brought before the House
-	\item Reside on the House
```
Comment (rhochgraf21) [inline constitution.tex:201]: 👍  (is covered later)

File: constitution.tex [hunk 3]
```diff
@@ -198,12 +198,9 @@
...
-	\item Hold an Executive Board office on the House
+	\item Hold an E-Board position
```
Comment (rhochgraf21) [inline constitution.tex:201]: 👍

File: constitution.tex [hunk 4]
```diff
@@ -198,12 +198,9 @@
...
 	\item Use Computer Science House facilities
-	\item Attend Computer Science House functions
-	\item Receive priority for available housing on the House when returning from cooperative education
```
Comment (rhochgraf21) [inline constitution.tex:205]: the (above line, about co-op return) seems important - can we work it into the below statement?

also might be good to mention "guaranteed housing" ... imo, the language in the below sentence may help back us up when talking to housing/reslife

Comment (esoccoli) [reply constitution.tex:205]: If i am remembering correctly, we don't guarantee housing, but we do give priority housing.

Comment (rhochgraf21) [reply constitution.tex:205]: after discussion  👍 

is removing attend csh functions a space thing?

Comment (esoccoli) [reply constitution.tex:205]: No, it is because the majority of CSH events are not restricted only to members

**Review by Qelxiros — APPROVED** [review]
lgtm

**Pull request merged**